tests/server/server-executeBulkWrite-004.phpt

```diff
@@ -0,0 +1,43 @@
+--TEST--
+MongoDB\Driver\Server::executeBulkWrite() with write concern (replica set secondary)
+--SKIPIF--
+<?php require __DIR__ . "/../utils/basic-skipif.inc"; ?>
+<?php NEEDS("REPLICASET"); CLEANUP(REPLICASET); ?>
+--FILE--
+<?php
+require_once __DIR__ . "/../utils/basic.inc";
+
+$manager = new MongoDB\Driver\Manager(REPLICASET);
+$server = $manager->selectServer(new MongoDB\Driver\ReadPreference(MongoDB\Driver\ReadPreference::RP_SECONDARY));
+
+$bulk = new MongoDB\Driver\BulkWrite();
+$bulk->insert(array('wc' => 0));
+
+$result = $server->executeBulkWrite(NS, $bulk, new MongoDB\Driver\WriteConcern(0));
+var_dump($result->isAcknowledged());
+var_dump($result->getInsertedCount());
+
+$writeConcerns = array(1, 2, MongoDB\Driver\WriteConcern::MAJORITY);
```

Comment:
That's tested on line 16. I separated it because an unacknowledged write concern does not throw an exception.